tests/python/test_optimization_passes.py

```diff
@@ -141,3 +141,42 @@ def nvfuser_fusion_id4(fd: FusionDefinition) -> None:
     ]
     # TODO: I wish we have an easy way for validation like in cpp tests.
     fd.execute(inputs)
+
+
+# https://github.com/NVIDIA/Fuser/issues/3369
+# don't need to replace constants in the same Id set
+def test_extent_substitution():
```

Comment:
Nit: since this doesn't explicitly test extent substitution, maybe we should rename it `test_square_linear` or something along those lines.

Reply:
changed